<launch>
    <rosparam command="load" file="$(find robosub)/param/joystick_driver.param" />
    <node name="joystick_driver" pkg="robosub" type="joystick_driver" />
</launch>
